File: official/docs/java/endShipper/CreateEndShipper.java
import com.easypost.EasyPost;

public class CreateEndShipper {
    public static void main(String[] args) {
        EasyPost.apiKey = System.getenv("EASYPOST_API_KEY");

        Map<String, Object> endShipperData = new HashMap<String, Object>();

        endShipperData.put("name", "FOO BAR");
        endShipperData.put("company", "BAZ");
        endShipperData.put("street1", "164 TOWNSEND STREET UNIT 1");
        endShipperData.put("street2", "UNIT 1");
        endShipperData.put("city", "SAN FRANCISCO");
        endShipperData.put("state", "CA");
        endShipperData.put("zip", "94107");
        endShipperData.put("country", "US");
        endShipperData.put("phone", "555-555-5555");
        endShipperData.put("email", "FOO@EXAMPLE.COM");

        EndShipper endShipper = EndShipper.create(endShipperData);

        System.out.println(endShipper);
    }
}

File: official/docs/php/end_shipper/CreateEndShipper.php
<?php

\EasyPost\EasyPost::setApiKey($_ENV['EASYPOST_API_KEY']);

$endshipper_params = array(
    "name" => "FOO BAR",
    "company" => "BAZ",
    "street1" => "164 TOWNSEND STREET UNIT 1",
    "street2" => "UNIT 1",
    "city" => "SAN FRANCISCO",
    "state" => "CA",
    "zip" => "94107",
    "country" => "US",
    "phone" => "555-555-5555",
    "email" => "FOO@EXAMPLE.COM"
);

$endshipper = \EasyPost\EndShipper::create($endshipper_params);

echo $endshipper;
